### PR TITLE
Added some Python documentation

### PR DESCRIPTION
**Pull request by hakonhagland**
Moved this PR from opm-common, see https://github.com/OPM/opm-common/pull/3667

### CODE REVIEW

**Comment by hakonhagland** [comment]
jenkins build this please

File: python/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+from opm.io.schedule import Schedule
+from opm.io.summary import SummaryConfig
+
+os.chdir("SPE1CASE1")
+deck  = Parser().parse('SPE1CASE1.DATA')
+state = EclipseState(deck)
+schedule = Schedule( deck, state )
+summary_config = SummaryConfig(deck, state, schedule)
+unit_system = deck.active_unit_system()
+
```
Comment (totto82) [inline python/README.md:54]: The unit_system is not used. Maybe more relevant for rates than porosity.

Comment (hakonhagland) [reply python/README.md:53]: Yes, I think we can just remove that line. Updated the PR with this change.

**Comment by totto82** [comment]
jenkins build this please

**Pull request merged**